dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/utils/DateUtilsTest.java

```diff
@@ -297,4 +297,5 @@ public void testTimeStampToDate() {
         Date date = DateUtils.timeStampToDate(0L);
         Assert.assertNull(date);
     }
+
```

Comment:
Should we remove this unrelated change here?

Reply:
sure. done